partition_gpu/partition_gpu.go

```diff
@@ -233,15 +233,15 @@ func cleanupAllGPUPartitions() error {
 	args := []string{"mig", "-dci"}
 	glog.Infof("Running %s %s", *nvidiaSmiPath, strings.Join(args, " "))
 	out, err := exec.Command(*nvidiaSmiPath, args...).Output()
-	if err != nil && !strings.Contains(string(out), "No GPU instances found") {
+	if err != nil && !strings.Contains(string(out), "No compute instances found") {
```

Comment:
No need to remove `No GPU instance found` check, older version of `nvidia-smi` still use it.